tests/e2e/run.sh

```diff
@@ -25,7 +25,7 @@ if [ "$#" -eq 0 ]; then
   # NOTE: If you want to run all tests under specific folder/section just
   # replace the dot (root directory below) with the folder name, e.g.
   # `--workloads-root-directory replication`.
-  python3 runner.py --workloads-root-directory .
+  python3 runner.py --workloads-root-directory "$SCRIPT_DIR/../../build"
```

Comment:
Why has this changed?

Reply:
https://github.com/memgraph/memgraph/pull/1550#discussion_r1452583018